contribute.njk

```diff
@@ -51,52 +38,13 @@
         </p>
         <h5>What happens next?</h5>
         <p>The issue will be reviewed by and incorporated into DataPortals.org pending no data errors.</p>
-        <h4>How to Add a Portal Using
-          <a href="https://explorer.okfnlabs.org/">Data Explorer</a>
-        </h4>
-        <p>
-          <em>Coming soon...</em>
-        </p>
```

Comment:
https://explorer.okfnlabs.org/#okfn/dataportals.org/edit/data/portals.csv didn't load for me 🤷